src/app/app.component.ts

```diff
@@ -20,6 +20,15 @@ import { MediaMatcher } from '@angular/cdk/layout';
 import { APP_ROUTES } from '../examples/example.module';
 import { Location } from '@angular/common';
 
+export function regexpFromArray(array, separator): RegExp {
```

Comment:
remove export define types for array and separator

`array` is a bad name 
separator consider to change to `delimiter`